### ISSUE TITLE
Гибридный режим STA+AP

### ISSUE THREAD

**Issue by da3020** [GyverLibs/GyverPortal#79]
Здравствуйте.
А получится сделать так: в основном режиме модуль висит в домашней инфраструктуре wifi как станция, но если вдруг необходимо, то,  чтобы не залазя на мачту можно было бы попасть в веб интерфейс напрямую через софтовую точку доступа модуля?
Спасибо.

**Comment by GyverLibs** [comment]
Это режим работы есп WIFI_AP_STA

**Comment by da3020** [comment]
Режим то понятен. ...то есть веб-интерфейс может параллельно выводится и клиенту AP модуля и клиенту, подключенному к локальной инфраструктуре wifi? Прям одновременно? То есть вышеописанная схема реализуема "из коробки"?
Просто в документации вижу, что в стартовых настройках  включается жёстко либо режим станции либо точки доступа.

**Comment by GyverLibs** [comment]
В стартовых настройках не указывается режим работы, всё будет работать из коробки в AP_STA

**Comment by da3020** [comment]
Спасибо. Тогда приступаю к практике.
...и последний вопрос, так сказать, чтоб добить ситуацию - реально ли клиентам подключённым к AP и клиентам, подключённым через STA отдавать РАЗНЫЕ  веб-интерфейсы?

**Comment by GyverLibs** [comment]
нет, но можно поднять два вебфейса на разных портах. По стандарту сайт открывается на 80 порту, можно поднять второй портал на любом другом (например 12345) и заходить на него как http://x.x.x.x:12345/
